Sources/HTTP/SimpleSocket/SimpleSocketSimpleServer.swift

```diff
@@ -8,53 +8,31 @@
 
 import Dispatch
 import Foundation
-#if os(Linux)
-import Signals
-#endif
-import Socket
+
 
 // MARK: Server
 
 /// An HTTP server that listens for connections on a TCP socket and spawns Listeners to handle them.
-public class BlueSocketSimpleServer: CurrentConnectionCounting {
-    /// Socket to listen on for connections
-    private let serverSocket: Socket
+public class SimpleSocketSimpleServer: CurrentConnectionCounting {
+    /// SimpleServerSocket to listen on for connections
+    private let serverSocket: SimpleServerSocket = SimpleServerSocket()
 
     /// Collection of listeners of sockets. Used to kill connections on timeout or shutdown
     private var connectionListenerList = ConnectionListenerCollection()
 
     // Timer that cleans up idle sockets on expire
-    private let pruneSocketTimer: DispatchSourceTimer
+    private let pruneSocketTimer: DispatchSourceTimer = DispatchSource.makeTimerSource(queue: DispatchQueue(label: "pruneSocketTimer"))
 
     /// The port we're listening on. Used primarily to query a randomly assigned port during XCTests
     public var port: Int {
         return Int(serverSocket.listeningPort)
     }
 
     /// Tuning parameter to set the number of queues
-    private var queueMax: Int
+    private var queueMax: Int = 4 //sensible default
```

Comment:
Why is this sensible?

Reply:
Because experimentally (with BlueSocket, so I might need to change it once I run this implementation through load testing), that's the smallest number I could use and still pull a webpage in a browser while a https://github.com/carlbrown/SwiftServerComparison run was in progress.